tests/test_pfs.py

```diff
@@ -39,51 +39,45 @@ def test_delete_non_existent_repo():
     assert len(client.list_repo()) == orig_repo_count
 
 
-# This test currently fails on Pachyderm v2.0.0-alpha.12 (see
-# https://github.com/pachyderm/pachyderm/issues/6156), but should be re-enabled
-# once that issue is fixed.
-# def test_delete_all_repos():
-#    client = python_pachyderm.Client()
-#
-#    util.create_test_repo(client, "test_delete_all_repos", prefix="extra-1")
-#    util.create_test_repo(client, "test_delete_all_repos", prefix="extra-2")
-#    assert len(client.list_repo()) >= 2
-#
-#    client.delete_all_repos()
-#    assert len(client.list_repo()) == 0
+def test_delete_all_repos():
+    client = python_pachyderm.Client()
+
+    util.create_test_repo(client, "test_delete_all_repos", prefix="extra-1")
+    util.create_test_repo(client, "test_delete_all_repos", prefix="extra-2")
+    assert len(client.list_repo()) >= 2
+
+    client.delete_all_repos()
+    assert len(client.list_repo()) == 0
 
 
 def test_start_commit():
     client, repo_name = sandbox("start_commit")
 
     commit = client.start_commit(repo_name, "master")
-    assert commit.repo.name == repo_name
-
-    commit = client.start_commit(repo_name, None)
```

Comment:
Just curious: it's a server-side change that `master` is no longer assumed?

Reply:
Yea, the server now requires branch name.

https://github.com/pachyderm/pachyderm/blob/65eaab4358a8af6b11aa783ab66c6f8c7c0d4db9/src/server/pfs/server/driver.go#L532